houston/ardu/copter/state.py

```diff
@@ -1,33 +1,56 @@
 __all__ = ['State']
 
+from ..connection import MAVLinkMessage, MAVLinkConnection
 from ...state import State as BaseState
 from ...state import var
 
+# These are the messages we should later listen to
+IMPORTANT_MESSAGE_NAMES = [
+    'GLOBAL_POSITION_INT',
+    'ATTITUDE',
+    'VFR_HUD',
+    'SYS_STATUS',
+    'EKF_STATUS_REPORT',
+    'HEARTBEAT',
+    'HOME_POSITION',
+]
+
 
 class State(BaseState):
     home_latitude = var(float, lambda c: -35.362938)  # TODO: fixed
     home_longitude = var(float, lambda c: 149.165085)  # TODO: fixed
     altitude = var(float,
-                   lambda c: c.connection.location.global_relative_frame.alt,  # noqa: pycodestyle
+                   lambda c: c.conn.location.global_relative_frame.alt,  # noqa: pycodestyle
```

Comment:
Yes. This is only there because of dronekit. Updating the state based on just the messages is a pain. If we want to get away from this, we basically need to implement whatever dronekit is doing right now. Some of them are easy to do but dome properties are more complicated like `flight_mode`.